src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/1.sql

```diff
@@ -1336,6 +1321,82 @@ AS
     COMMIT TRANSACTION
 GO
 
+--
+-- STORED PROCEDURE
+--     Acquires export jobs.
+--
+-- DESCRIPTION
+--     Timestamps the available export jobs and sets their statuses to running.
+--
+-- PARAMETERS
+--     @expirationDateTime
+--         * The time before which an export job is considered stale
+--     @maximumNumberOfConcurrentJobsAllowed
+--         * The maximum number of running jobs we can have at once
+--     @heartbeatDateTime
```

Comment:
It's not a big deal here, but you could use the SQL functions to determine the current time instead of passing in times. (you can still parameterize the expiration timespans). The reason would be that different FHIR servers can each have differing clock skews.

Reply:
Good idea. Switched to `SYSUTCDATETIME()` in 5633f4a.

Reply:
I think you should try to stick with all datetimes on the SQL side. You are still taking UtcNow on the FHIR server and passing it in to the stored procedure.

Reply:
Right, thanks. Fixed in d9a5f4b.